<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PrimerFinderDialog</class>
 <widget class="QDialog" name="PrimerFinderDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>760</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>900</width>
    <height>760</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>900</width>
    <height>760</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTextBrowser" name="textBrowser">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>380</y>
     <width>641</width>
     <height>361</height>
    </rect>
   </property>
  </widget>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>720</y>
     <width>166</width>
     <height>29</height>
    </rect>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>144</width>
     <height>324</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="title">
       <string>Amplicon Length</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_6">
       <item row="0" column="0">
        <layout class="QGridLayout" name="gridLayout_5">
         <item row="0" column="0">
          <widget class="QLabel" name="minAmpliconLengthLabel">
           <property name="text">
            <string>Minimum</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QSpinBox" name="spinBox"/>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="maxAmpliconLengthLabel">
           <property name="text">
            <string>Maximum</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QSpinBox" name="spinBox_2"/>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>218</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>0</y>
     <width>501</width>
     <height>421</height>
    </rect>
   </property>
   <property name="title">
    <string>Primer Characteristics</string>
   </property>
   <widget class="QGroupBox" name="groupBox_6">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>20</y>
      <width>311</width>
      <height>121</height>
     </rect>
    </property>
    <property name="title">
     <string>5' Specifications</string>
    </property>
    <widget class="QLineEdit" name="forward5Addition">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>30</y>
       <width>131</width>
       <height>25</height>
      </rect>
     </property>
     <property name="inputMethodHints">
      <set>Qt::ImhPreferUppercase</set>
     </property>
     <property name="maxLength">
      <number>10</number>
     </property>
    </widget>
    <widget class="QLineEdit" name="reverse5Addition">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>80</y>
       <width>131</width>
       <height>25</height>
      </rect>
     </property>
     <property name="inputMethodHints">
      <set>Qt::ImhPreferUppercase</set>
     </property>
     <property name="inputMask">
      <string>&gt;; </string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="maxLength">
      <number>0</number>
     </property>
    </widget>
    <widget class="QLabel" name="forward5AdditionLabel">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>121</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>Forward Primer</string>
     </property>
    </widget>
    <widget class="QLabel" name="forward5AdditionLabel_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>80</y>
       <width>121</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>Reverse Primer</string>
     </property>
    </widget>
    <widget class="QToolButton" name="addForwardRestrictionSiteButton">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>30</y>
       <width>26</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
    <widget class="QToolButton" name="addReverseRestrictionSiteButton">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>80</y>
       <width>26</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_5">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>160</y>
      <width>290</width>
      <height>201</height>
     </rect>
    </property>
    <property name="title">
     <string>3' Specifications</string>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>40</y>
       <width>290</width>
       <height>155</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="title">
         <string>Forward Primer</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="0" column="1">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>1</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>2</string>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>3</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>A</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QRadioButton" name="forward1AradioButton">
           <property name="text">
            <string/>
           </property>
           <property name="autoExclusive">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QRadioButton" name="forward2AradioButton">
           <property name="text">
            <string/>
           </property>
           <property name="autoExclusive">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="3">
          <widget class="QRadioButton" name="forward3AradioButton">
           <property name="text">
            <string/>
           </property>
           <property name="autoExclusive">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>T</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QRadioButton" name="forward1TradioButton">
           <property name="text">
            <string/>
           </property>
           <property name="autoExclusive">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="QRadioButton" name="forward2TradioButton">
           <property name="text">
            <string/>
           </property>
           <property name="autoExclusive">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="2" column="3">
          <widget class="QRadioButton" name="forward3TradioButton">
           <property name="text">
            <string/>
           </property>
           <property name="autoExclusive">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string>C</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QRadioButton" name="forward1GradioButton">
           <property name="text">
            <string/>
           </property>
           <property name="autoExclusive">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="3" column="2">
          <widget class="QRadioButton" name="forward2GradioButton">
           <property name="text">
            <string/>
           </property>
           <property name="autoExclusive">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="3" column="3">
          <widget class="QRadioButton" name="forward3GradioButton">
           <property name="text">
            <string/>
           </property>
           <property name="autoExclusive">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label_10">
           <property name="text">
            <string>G</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QRadioButton" name="forward1CradioButton">
           <property name="text">
            <string/>
           </property>
           <property name="autoExclusive">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="4" column="2">
          <widget class="QRadioButton" name="forward2CradioButton">
           <property name="text">
            <string/>
           </property>
           <property name="autoExclusive">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="4" column="3">
          <widget class="QRadioButton" name="forward3CradioButton">
           <property name="text">
            <string/>
           </property>
           <property name="autoExclusive">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_4">
        <property name="title">
         <string>Reverse Primer</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="1">
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>1</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>2</string>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>3</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_14">
           <property name="text">
            <string>A</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QRadioButton" name="reverse1AradioButton">
           <property name="text">
            <string/>
           </property>
           <property name="autoExclusive">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QRadioButton" name="reverse2AradioButton">
           <property name="text">
            <string/>
           </property>
           <property name="autoExclusive">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="3">
          <widget class="QRadioButton" name="reverse3AradioButton">
           <property name="text">
            <string/>
           </property>
           <property name="autoExclusive">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_12">
           <property name="text">
            <string>T</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QRadioButton" name="reverse1TradioButton">
           <property name="text">
            <string/>
           </property>
           <property name="autoExclusive">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="QRadioButton" name="reverse2TradioButton">
           <property name="text">
            <string/>
           </property>
           <property name="autoExclusive">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="2" column="3">
          <widget class="QRadioButton" name="reverse3TradioButton">
           <property name="text">
            <string/>
           </property>
           <property name="autoExclusive">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_11">
           <property name="text">
            <string>C</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QRadioButton" name="reverse1GradioButton">
           <property name="text">
            <string/>
           </property>
           <property name="autoExclusive">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="3" column="2">
          <widget class="QRadioButton" name="reverse2GradioButton">
           <property name="text">
            <string/>
           </property>
           <property name="autoExclusive">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="3" column="3">
          <widget class="QRadioButton" name="reverse3GradioButton">
           <property name="text">
            <string/>
           </property>
           <property name="autoExclusive">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label_13">
           <property name="text">
            <string>G</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QRadioButton" name="reverse1CradioButton">
           <property name="text">
            <string/>
           </property>
           <property name="autoExclusive">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="4" column="2">
          <widget class="QRadioButton" name="reverse2CradioButton">
           <property name="text">
            <string/>
           </property>
           <property name="autoExclusive">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="4" column="3">
          <widget class="QRadioButton" name="reverse3CradioButton">
           <property name="text">
            <string/>
           </property>
           <property name="autoExclusive">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QLabel" name="minimumLabel">
    <property name="geometry">
     <rect>
      <x>73</x>
      <y>32</y>
      <width>69</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Min</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="maxTmSpinBox">
    <property name="geometry">
     <rect>
      <x>149</x>
      <y>57</y>
      <width>47</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="maximumLabel">
    <property name="geometry">
     <rect>
      <x>149</x>
      <y>32</y>
      <width>75</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Max</string>
    </property>
   </widget>
   <widget class="QLabel" name="tmLabel">
    <property name="geometry">
     <rect>
      <x>13</x>
      <y>57</y>
      <width>24</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Tm</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="tmLabel_2">
    <property name="geometry">
     <rect>
      <x>13</x>
      <y>89</y>
      <width>54</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Length</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QSpinBox" name="minPrimerLengthSpinBox">
    <property name="geometry">
     <rect>
      <x>73</x>
      <y>89</y>
      <width>47</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="maxPrimerLengthSpinBox">
    <property name="geometry">
     <rect>
      <x>149</x>
      <y>89</y>
      <width>47</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="minTmSpinBox">
    <property name="geometry">
     <rect>
      <x>73</x>
      <y>57</y>
      <width>47</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
